ubuntu/standard/Dockerfile.template

```diff
@@ -80,6 +80,15 @@ RUN curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /et
 # Run dotnet command to populate initial cache
 RUN dotnet help
 
+# Configure AzCLI 2.0  Instructions taken from https://docs.microsoft.com/en-us/cli/azure/install-azure-cli
+RUN echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheezy main" | \
+     tee /etc/apt/sources.list.d/azure-cli.list \
+     && apt-key adv --keyserver packages.microsoft.com --recv-keys 417A0893 \
+     && apt-get install -y --no-install-recommends apt-transport-https \
```

Comment:
Please follow the formatting for multi-line RUN commands by adding only a single character before the "&&" (this lines up the commands being executed in sequence for easier readability).

Reply:
done